.env

```diff
@@ -24,6 +24,9 @@ HASH_GRAPH_PG_USER=graph
 HASH_GRAPH_PG_PASSWORD=graph
 HASH_GRAPH_PG_DATABASE=graph
 
+HASH_GRAPH_REALTIME_PG_USER=realtime
```

Comment:
Yes, we explicitly subscribe to `HASH_GRAPH_PG_DATABASE` for updates

Reply:
I guess what I was asking is, is REALTIME part of the Graph? Like we don't say HASH_GRAPH_WORKSPACE_...
even though the workspace explicitly subscribes to the graph.

Anyway I'm fine with this

Reply:
I suppose it would be crossing some boundary making it part of the graph rather than the workspace. It makes no assumptions on the underlying data, it maps directly to the raw database representation. We might revisit what we need from the change events when collab subscribes to this new redis feed